File: ModernWebStore/ModernWebStore.Domain/Entities/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ModernWebStore.Domain.Entities
{
    public class Cliente
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Rua { get; set; }
        public string Numero { get; set; }
        public string Bairro { get; set; }
        public string CEP { get; set; }
        public string Celular { get; set;}
        public string Telefone { get; set; }
        public string Observacao { get; set; }
        public string CPF { get; set; }
        public string CNPJ { get; set; }
        
        public ICollection<OrdemServico> OrdemServicos { get; set; }

    }
}

File: KTAX_SOS_Workflow.WebApp/app/models/api-models/Cliente.ts
/// <reference path="../../configs/_all.ts" />
module Consermaq {
    export class Cliente {  
        public id: number;
        public nome: string;
        public rua: string;
        public numero: string;
        public bairro: string;
        public observacao: string;
        public cep: string;
        public celular: string;
        public telefone: string;
  
        public cpf: string;
        public cnpj: string;

        public ordemServicos: Array<OrdemServico>;

        constructor() {        
        }
    }
}
